examples/js/loaders/TDSLoader.js

```diff
@@ -22,7 +22,7 @@ THREE.TDSLoader = function ( manager ) {
 	this.materials = [];
 	this.meshes = [];
 
-	this.path = "";
+	this.path = undefined;

```

Comment:
This can be deleted.

Reply:
In theory you are right.
But I normally want to keep track of my properties so I like to declare them at the top even when I set them to undefined. So I can always check the top what properties/variables are available on the object, but that is personal preference.
If you rather delete it I will do, but I would prefer to keep it

Reply:
It would be nice if `TDSLoader` looks a little bit more like `GLTFLoader ` or `ColladaLoader`. I understand your point but deleting the line results in a code that is more consistent to other loaders.